### ISSUE TITLE
False positive: FM not being used: variable not being used is a FM and it is being used

### ISSUE THREAD

**Issue by nununo** [abaplint/abaplint#2071]
Hi,

[This](https://github.com/abapinho/abapTimeMachine/pull/109#discussion_r680173776) check is reporting that:

> Variable "z_timem_object_name_shelp" not used

Two issues:
- `Z_TIMEM_OBJECT_NAME_SHELP` is a function module, not a variable
- It is being used in search help `ZTIMEM_OBJECT_NAME`

Cheers!

**Comment by larshp** [comment]
@nununo try again, it should now report the correct name of the variable

**Comment by nununo** [comment]
@larshp,

I just tried re-running it and indeed now it's mentioning specific variable `shlp` not being used. Good job!

It still suggests that I should delete the FM though 😄 

```
+FUNCTION z_timem_object_name_shelp.
Variable "shlp" not used

Suggest following fix,

⬇️ Suggested change
-FUNCTION z_timem_object_name_shelp.
```

**Comment by larshp** [comment]
heh, yea, the function module parameters are a bit special, as its not part of the code

**Comment by larshp** [comment]
@nununo try again, 2.78.8 published

**Comment by larshp** [comment]
closing, no reply
